repository/t.json

```
@@ -2762,6 +2762,17 @@
 				}
 			]
 		},
+		{
+			"name": "TodoTxt",
+			"details": "https://github.com/overflowy/todotxt",
+			"labels": ["todotxt, todotxt syntax", "tasks", "task manager"],
+			"releases": [
+				{
+					"sublime_text": "*",
+					"branch": "master"
```

Well, that took some figuring out from the CI output 😅 Your default branch is actually "main" not "master. But you're using tags, soooo ...

```suggestion
					"tags": true
```